integration/fuse/bin/alluxio-fuse

```diff
@@ -109,7 +109,8 @@ fuse_stat() {
   fi
   local fuse_info=$("${jps}" | grep AlluxioFuse)
   if [[ -z ${fuse_info} ]]; then
-    echo "AlluxioFuse: not running"
+    echo "No running AlluxioFuse process is found."
```

Comment:
It is simpler to say `AlluxioFuse is not running`

Reply:
done